samples/sampler/stories/default/Popup.js

```diff
@@ -23,6 +23,7 @@ storiesOf('Agate', module)
 					scrimType={select('scrimType', ['none', 'translucent', 'transparent'], Config, 'translucent')}
 					spotlightRestrict={select('spotlightRestrict', ['self-first', 'self-only'], Config, 'self-only')}
 					title={text('title', Config, 'Title')}
+					center={boolean('center', Config)}
```

Comment:
By using the `Config`, the storybook tries to get the default value of the `center` defined in the `Popup/Popup.js`. But the default value of the `center` is not defined.

Reply:
Done. Thank you!